package-lock.json

```diff
@@ -1,31 +1,4559 @@
 {
   "name": "@github/template-parts",
   "version": "0.3.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Are we ready to commit to npm 7?

Reply:
I think so. We need no features in npm 6 that I know of, and we've already updated other packages. dotcom still needs npm 6 for some pre and post hooks that went missing from npm 6 to npm 7.